src/encoded/tests/test_aggregation.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
-from .workbook_fixtures import app_settings, workbook
 from ..utils import delay_rerun
+from .workbook_fixtures import app_settings, app, workbook
```

Comment:
Codacy found an issue: [Unused app_settings imported from workbook_fixtures](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5767737)

Reply:
Fixed.

Reply:
Worth noting on these that it is _very important_ not to remove these, so I added `notice_pytest_fixture` to make it not seem unused.

Reply:
I have also opened a bug report in PyCharm, which should not be mentioning these. It shows poor support for `pytest`. I suppose Codacy also needs a bug report at some point.